src/main/java/com/exadel/backendservice/controllers/EventController.java

```diff
@@ -0,0 +1,42 @@
+package com.exadel.backendservice.controllers;
+
+import com.exadel.backendservice.dto.EventWithLabelAndDirectionDto;
+import com.exadel.backendservice.services.EventService;
+import com.exadel.backendservice.services.impl.UserServiceImpl;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/event/")
+@Api(tags = "Сервис для работы с анонсами событий")
+public class EventController {
+    private static final Logger LOGGER = LoggerFactory.getLogger(UserServiceImpl.class);
+
+    private final EventService eventService;
+
+    @Autowired
```

Comment:
no need to explicitly inject EventService, if @RequiredArgsConstructor is set

Reply:
corrected